src/index.html

```diff
@@ -1,16 +1,45 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible"
+      content="ie=edge">
     <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet"
+      href="./style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="#"
+      method="post"
+      data-qa="big"
+      class="form">
+      <img src="images/Search.svg"
+        alt="magnifier"
+        class="big-img">
+      <input type="text"
+        class="input big-input"
+        name="big-serch-bar"
+        data-qa="keypress"
+        placeholder="Try &#34Los Angeles&#34">
+    </form>
+
+    <form action="#"
+      method="post"
+      data-qa="small"
+      class="form">
+      <img src="images/Search.svg"
+        alt="magnifier"
+        class="small-img">
+      <input type="text"
+        class="input small-input"
+        name="small-serch-bar"
+        placeholder="Try &#34Los Angeles&#34">
+    </form>
+
   </body>
-</html>
+
+</html>
```

Comment:
Don't forget to add a newline at the end
https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file